README.md

```diff
@@ -27,7 +27,17 @@ fn get_contents(&mut self) -> Result<String, Box<Error>>;
 fn set_contents(&mut self, String) -> Result<(), Box<Error>>;
 ```
 
-`ClipboardContext` is a type alias for one of {`WindowsClipboardContext`, `OSXClipboardContext`, `X11ClipboardContext`, `NopClipboardContext`}, all of which implement `ClipboardProvider`. Which concrete type is chosen for `ClipboardContext` depends on the OS (via conditional compilation).
+`ClipboardContext` is a type alias for one of 
+{
+
+- `WindowsClipboardContext`
+- `AndroidClipboardContext`
+- `OSXClipboardContext`
+- `X11ClipboardContext`
+- `NopClipboardContext`
+
+}
+,all of which implement `ClipboardProvider`. Which concrete type is chosen for `ClipboardContext` depends on the OS (via conditional compilation).
```

Comment:
The formatting of this is all over the place (see rendered readme).

I'd suggest a leading sentence like "`ClipboardContext` is a type alias for one of the following types implementing `Clipboard Provider`. Which concrete type is chosen for `ClipboardContext` depends on the OS.", then add the list of types as bullet-points without any weird brackets.